index.js

```diff
@@ -523,8 +530,8 @@ class CreateCertificatePlugin {
     }
 
     return this.listCertificates()
-      .then(({ CertificateSummaryList }) => {
-        let cert = CertificateSummaryList.filter(({ DomainName }) => DomainName == domainName)[0];
+      .then((certificates) => {
```

Comment:
```suggestion
      .then(certificates => {
```

nit: we can drop these